app/components/ch/components/Pagination.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <button type="button" :class="classes">
+  <button type="button" :class="classes" aria-label="pagination">
```

Comment:
here, replace the pagination with something meaningful for screenreaders, «page 1», «page two» «next page» etc